src/core/Object3D.js

```diff
@@ -659,6 +659,7 @@ Object3D.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 		if ( this.renderOrder !== 0 ) object.renderOrder = this.renderOrder;
 		if ( JSON.stringify( this.userData ) !== '{}' ) object.userData = this.userData;
 
+		object.layers = { mask: this.layers.mask };
```

Comment:
Actually, considering how stable that class is, I think I would just do this:

```js
object.layers = this.layers.mask;
```

Reply:
Done.